src/AwsCryptographicMaterialProviders/Defaults.dfy

```diff
@@ -14,16 +14,16 @@ module {:extern "Defaults"} Defaults {
     ensures
       commitmentPolicy == Crypto.FORBID_ENCRYPT_ALLOW_DECRYPT
       ==>
-      res == Crypto.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384
+        res == Crypto.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384
 
     ensures
       || commitmentPolicy == Crypto.REQUIRE_ENCRYPT_ALLOW_DECRYPT
       || commitmentPolicy == Crypto.REQUIRE_ENCRYPT_REQUIRE_DECRYPT
       ==>
-      res == Crypto.ALG_AES_256_GCM_HKDF_SHA512_COMMIT_KEY_ECDSA_P384
+        res == Crypto.ALG_AES_256_GCM_HKDF_SHA512_COMMIT_KEY_ECDSA_P384
   {
     if commitmentPolicy == Crypto.FORBID_ENCRYPT_ALLOW_DECRYPT then
       Crypto.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384 else
```

Comment:
If we formatted this as:
```suggestion
    if commitmentPolicy == Crypto.FORBID_ENCRYPT_ALLOW_DECRYPT
      then Crypto.ALG_AES_256_GCM_IV12_TAG16_HKDF_SHA384_ECDSA_P384
      else Crypto.ALG_AES_256_GCM_HKDF_SHA512_COMMIT_KEY_ECDSA_P384
```
Would that avoid the big space proposed?

Reply:
Yes, exactly. Would you like me to apply manually this change as part of this PR?